### DISCUSSION TITLE
Can free Orgs have external collaborators in private repositories?

### DISCUSSION THREAD

**Discussion by gtirloni** [community/community#46538] [New to GitHub] [answered]
If I have a free organization (i.e. not on a paid plan of any kind) and I have private repositories in that org.

Am I able to invite collaborators to those private repositories?

EDIT: https://github.blog/changelog/2020-04-14-private-repositories-with-unlimited-collaborators-available-to-all-github-accounts-and-changes-to-github-paid-plans/

**Comment by queenofcorgis** [comment, marked as answer]
Adding an outside collaborator to a private repository will not use one of your paid licenses - check out our [docs](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization#about-outside-collaborators) for more information. 

_edited for clarity with details included below_

**Reply by gtirloni** [reply]
Even for Free Orgs? What license would be consumed in that case?

**Reply by queenofcorgis** [reply]
Apologies - I checked in on this - on a free plan there would **not** be any paid seats required. I'll update our docs too so this is more clear. 

The difference is that private repos have advanced features on the upgraded plans, and they do not on Free. There are actually no seats required at all on Free, it’s unlimited…. until/unless you upgrade to a paid plan, and then you will want to look at your list of members and collaborators with access to private repos.

**Reply by Buddhist-Ham** [reply]
Hi, care to lete Know whether is there a problem?  Is the project be held back for certain reason? Drop me an email or WhatsApp me at +60163396963 (Malaysian Number: Andy
